### ISSUE TITLE
trigger liquid-spacer height calculations

### ISSUE THREAD

**Issue by ktrhn** [ember-animation/liquid-fire#445]
In the current state liquid-spacer is not fluid, i.e. if the width of the outer element (or the screen) changes, the calculated and set width does not adapt. Is there some way to trigger "rerendering" the liquid-spacer? Unfortunately manually setting the css width to auto does not help as liquid-spacer is broken afterwards.

**Comment by grapho** [comment]
I agree with this, at least at face value.  I am not sure how practical it is.  HTML will naturally adjust its height when flow content wraps to new lines or when a child element increases size of other reasons.  liquid-fire constrains this behavior in order to provide smoother animations artificially.  It sounds like you are talking about animating the natural transition at a browser level, which i dont think is currently possible.  Maybe someone else can confirm?

**Comment by grapho** [comment]
@ktrhn one workaround i use (for width only) is using css to force `width: 100% !important;` on the liquid spacer.  that will at least force the spacer to remain tat the width of its containing element.
